### ISSUE TITLE
Unable to convert any files.

### ISSUE THREAD

**Issue by itsfood** [ichttt/MCLang2Json#7]
If I hit Convert File button, it prompts me to select a file, but does nothing with it.

**Comment by ichttt** [comment]
Could you post the output when running the application from a console?

**Comment by itsfood** [comment]
I don’t see any console opening.

**Comment by ichttt** [comment]
That's right, you should run it from a cosole yourself using java -jar [Filename].

**Comment by byteduck** [comment]
It's because you didn't embed the JSON library into the jar.
